policy/modules/contrib/iscsi.if

```diff
@@ -208,3 +208,22 @@ interface(`iscsi_admin',`
 	files_search_tmp($1)
 	admin_pattern($1, iscsi_tmp_t)
 ')
+
+########################################
+## <summary>
+##	Read iscsi PID files.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`iscsi_read_pid_files',`
+	gen_require(`
+		type iscsi_var_run_t;
+	')
+
+	files_search_pids($1)
+	read_files_pattern($1, iscsi_var_run_t, iscsi_var_run_t)
```

Comment:
nit but i will mention it anyway. The applicable file context specification do not mention any directories with type iscsi_var_run_t and so this interface adds a redundant rule to the policy for each caller. It is inefficient but it also distracts when reading the policy because there is a inconsistency when comparing the file context with the rules expanded from this interface.

ie. i would use the read_file_perms permission set instead of the read_files_pattern pattern

Reply:
Thank you for the feedback. I considered using just the permission set and then chose pattern instead for the sake of consistency as almost every _read_pid_files interface uses pattern even when not necessary.